packages/components/src/ListBox/src/ListBox.tsx

```diff
@@ -0,0 +1,130 @@
+import { type StyledComponentProps, useStyledSystem, type ResponsiveProp, useResponsiveValue } from "@hopper-ui/styled-system";
+import { forwardRef, type ForwardedRef } from "react";
+import { useContextProps, ListBox as RACListBox, type ListBoxProps as RACListBoxProps, composeRenderProps } from "react-aria-components";
+
+import { DividerContext } from "../../Divider/index.ts";
+import { HeaderContext } from "../../Header/index.ts";
+import { SectionContext } from "../../Section/index.ts";
+import { composeClassnameRenderProps, SlotProvider, cssModule } from "../../utils/index.ts";
+
+import { ListBoxContext } from "./ListBoxContext.ts";
+import type { ListBoxItemSize } from "./ListBoxItem.tsx";
+import { ListBoxItemContext } from "./ListBoxItemContext.ts";
+import { useLoadOnScroll } from "./useLoadOnScroll.ts";
+
+import styles from "./ListBox.module.css";
+
+export const GlobalListBoxCssSelector = "hop-ListBox";
+
+export interface ListBoxProps<T> extends StyledComponentProps<Omit<RACListBoxProps<T>, "orientation | layout">> {
```

Comment:
FYI we might have to support orientation and layout at some point, i'm thinking for the ColorPicker

Reply:
alright, but for now, it's ok like this?

Reply:
yeah sure